### ISSUE TITLE
case with default not working (at least using JsExpressionHandler)

### ISSUE THREAD

**Issue by samogot** [neuland/jade4j#155]
code
```jade
  case tag_type
    when 'inline'
        //...
    when 'self_closing'
        //...
    default
        //...
```
throws
```
de.neuland.jade4j.exceptions.JadeCompilerException: unable to evaluate [tag_type == default] - ScriptException <eval>:1:12 Expected an operand but found default
tag_type == default
            ^ in <eval> at line number 1 at column number 12 in node_modules/bemto.jade/lib/bemto_custom_tag.jade:40
```

**Comment by chbloemer** [comment]
I can't reproduce your exception:
My code
```jade
- var friends = 10
case friends
    when 0
        p you have no friends
    when 1
        p you have a friend
    default
        p you have #{friends} friends

- var tagType = "default"
case tagType
    when "p"
        p you have no p
    when "br"
        p you have a br
    default
        p you have a #{tagType} tag
```
generates
```html

<p>you have 10 friends</p>
<p>you have a default tag</p>
```

**Comment by chbloemer** [comment]
Fixed in 1.2.6
